lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +207,34 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const container = resource.spec.url?.replace('oci://', '');
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: container,
+        };
+        if (resource.spec.ref?.digest) {
+          dep.currentDigest = resource.spec.ref.digest;
+          dep.datasource = DockerDatasource.id;
+          if (resource.spec.ref?.tag) {
+            logger.debug('A digest and tag was found, ignoring tag');
+          }
+        } else if (resource.spec.ref?.tag) {
+          const containerTag = getDep(
```

Comment:
```suggestion
          dep = getDep(
```
make dep a let instead of const, use same for digest above.

the get dep will do almost all the work below, just disable the auto-replace.

we'll add the registry aliases feature in a later PR.

Reply:
```
Type 'PackageDependency<Record<string, any>>' is not assignable to type 'PackageDependency<FluxManagerData>'.
Property 'components' is missing in type 'Record<string, any>' but required in type 'FluxManagerData'.
```

Any suggestion on a way around this?